src/db/works/works.service.ts

```diff
@@ -1,16 +1,24 @@
 import { Injectable } from '@nestjs/common';
 import { InjectModel } from '@nestjs/mongoose';
 import { Model } from 'mongoose';
+import { Library, types } from 'ffi-napi';
 
 import * as models from './models';
 import { UsersService } from '../users/users.service';
 
 @Injectable()
 export class WorksService {
+    
+    private wordCounterLib: any;
+
     constructor(
         @InjectModel('Work') private readonly workModel: Model<models.Work>,
         @InjectModel('Section') private readonly sectionModel: Model<models.Section>,
-        private readonly usersService: UsersService) {}
+        private readonly usersService: UsersService) {
+            this.wordCounterLib = new Library("../../native/compiled/word_counter", {
```

Comment:
This is another one of those things that should probably be centralized--I was thinking that we'd want all of our native modules to live in `/native/compiled/`, and soooome part of the build process generates them. That, or we build them by hand, and just leave that folder as something only seldom-touched.